clients/src/main/resources/common/message/ListConfigResourcesRequest.json

```diff
@@ -17,10 +17,14 @@
   "apiKey": 74,
   "type": "request",
   "listeners": ["broker"],
-  "name": "ListClientMetricsResourcesRequest",
-  "validVersions": "0",
+  "name": "ListConfigResourcesRequest",
+  // Version 0 is used as ListClientMetricsResourcesRequest which only lists client metrics resources.
+  // Version 1 adds ResourceTypes field (KIP-1142). If there is no specified ResourceTypes, it should return all configuration resources.
+  "validVersions": "0-1",
   "flexibleVersions": "0+",
   "fields": [
+    { "name": "ResourceTypes", "type": "[]int8", "versions": "1+", "ignorable": true,
```

Comment:
I don't see the value of `ignorable` here. If it's v0, the default is essentially an array containing `ConfigResource.CLIENT_METRICS`.